### DISCUSSION TITLE
http request uri is NULL

### DISCUSSION THREAD

**Discussion by Ptitjbus** [supabase/supabase#17490] [Questions]
Hi everyone ! 
I followed the documentation (https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs#bonus-profile-photos)  to delete avatar images in the storage when it's replaced but i've got an error says "**http_request.uri is NULL**" code 'XX000' when i try to update avatar_url

```
Request url : https://thnhwrmdzjwhgfjiofbk.supabase.co/rest/v1/profiles
Request Method:POST
Status Code: 500
Response : {
    "code": "XX000",
    "details": null,
    "hint": null,
    "message": "http_request.uri is NULL"
}
```
There are my functions 
![image](https://github.com/supabase/supabase/assets/54323824/c198d716-d3ee-405e-93d0-88b7ec8b118f)

Thanks for your help  !

**Comment by dutchdata** [comment]
I ran into this same issue and fixed it by updating the `next.config.js` file (see below). I got the answer by reading the error message returned by Next in local dev, which suggested this page: https://nextjs.org/docs/messages/next-image-unconfigured-host.

Hope this helps.

```
/** @type {import('next').NextConfig} */
const nextConfig = {}

module.exports = nextConfig

// * this allows images from supabase storage to be displayed on the site without error
module.exports = {
    images: {
      remotePatterns: [
        {
          protocol: 'https',
          hostname: '[your-project-id-or-supabase-subdomain].supabase.co',
          port: '',
          pathname: '/storage/**',
        },
      ],
    },
  }
  ```

**Reply by dutchdata** [reply]
*Your `hostname` would be `thnhwrmdzjwhgfjiofbk.supabase.co`.
